assets/js/app/LocusZoom.js

```diff
@@ -151,5 +210,13 @@
         return response.promise;
     };
 
+    exports.isString = function(x) {
```

Comment:
I don't think this method is necessary, and it's tough to argue this is a thing that the LocusZoom singleton should "do" (in other words the LocusZoom singleton does a lot of things like populate divs with instances or parse position queries, but type-checking is something that core JS should be doing).

In JavaScript the only time you'd need to check the value with `instanceof` is if the value is an object-wrapped string (instantiated with the `x = new String("foo");` syntax, which we should really never do). As such a simple `typeof x === "string"` is sufficient wherever we need to check for stringyness.